x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/load_action_error_log.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HttpSetup } from '@kbn/core/public';
+import type { SortOrder } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { escapeKuery } from '@kbn/es-query';
```

Comment:
I think this might be blowing up the bundle size, will look into it

Reply:
Yes, you need to do something like I did in my last PR to avoid blowing up the bundle size

Reply:
I can help you here